### ISSUE TITLE
Possibly breaks live-reload of addons

### ISSUE THREAD

**Issue by ef4** [ef4/ember-auto-import#99]
Any Ember addon can be live-reloaded in development via `isDevelopingAddon(){ return true }`. But [here is a report](https://discuss.emberjs.com/t/not-detected-file-changes-on-browser/15251/4?u=ef4) of the latest ember-auto-import breaking that.

**Comment by allthesignals** [comment]
I ran into this issue as well, took me an hour to track down. Any thoughts on a workaround?

**Comment by allthesignals** [comment]
Rollback version of ember-auto-import to 1.2.5

**Comment by urbany** [comment]
@ef4 after upgrading to 1.2.9 (only change I made) I can't build for production with sourcemaps anymore:
```Error: Build Canceled: Broccoli Builder ran into an error with `SourceMapConcat` plugin. 💥
ENOENT: no such file or directory, open '/.../tmp/source_map_concat-input_base_path-CcL2UhBq.tmp/assets/test-support.js'
```

Will debug more, not sure if is something on my side.

**Comment by ef4** [comment]
@urbany yeah, sorry about that, that's issue https://github.com/ef4/ember-auto-import/issues/105. Working on a fix.

**Comment by urbany** [comment]
@ef4 no need to apologise, thank you so much for this amazing addon! Will track that other issue, thanks!
